apps/src/componentLibrary/tabs/_TabPanel.tsx

```diff
@@ -6,15 +6,23 @@ interface TabPanelProps {
   id: string;
   /* The ID of the button element that controls this panel's visibility. */
   labelledBy: string;
+  className?: string;
```

Comment:
@levadadenys specifically want to run this change by you. This allows consumers to pass in a custom class name for the container that wraps this specific tab. I added this because I needed a way to assign `height: 100%` to the `div` that wrapped the tab. Let me know if you think this would be an issue. I checked other usages of `Tabs` and saw only one other (LandingPage.jsx) which seems to only use the tabs portion (not the content) so this shouldn't affect it.

Reply:
Adding this prop LGTM overall, thanks for adding!
In this specific case I think we can use my styles suggestion above, however for the future I think we should keep it in case we'll need some custom styles for specific tab/number of tabs(but not all) panels.

One small nit: Would be awesome if we also udpate Tabs Chagnelog here (can be [0.2.1]) just to keep logging the updates to hte component

Reply:
Sounds good! In that case, just to keep this change more targeted, I can revert the design system changes and use your suggestion directly. If you still think it's worthwhile having a custom class name for the tab element, I can split that off into a separate change.